From: Jan Kara <jack@suse.cz>
Subject: Warn if XFS realtime volumes are used
References: fate#314871
Patch-mainline: Never

Since XFS realtime volumes are not really first class feature, warn user if
they are used.

Signed-off-by: Jan Kara <jack@suse.cz>

diff -rupX /crypted/home/jack/.kerndiffexclude linux-3.0-SLE11-SP3/fs/xfs/xfs_rtalloc.c linux-3.0-SLE11-SP3-xfs-warn/fs/xfs/xfs_rtalloc.c
--- linux-3.0-SLE11-SP3/fs/xfs/xfs_rtalloc.c	2011-07-22 04:17:23.000000000 +0200
+++ linux-3.0-SLE11-SP3-xfs-warn/fs/xfs/xfs_rtalloc.c	2013-02-27 13:18:09.317872559 +0100
@@ -2238,6 +2238,9 @@ xfs_rtmount_init(
 		return EIO;
 	}
 	xfs_buf_relse(bp);
+	xfs_warn(mp, "Realtime volumes are an experimental feature. Please "
+		"contact SUSE Technical Services in case you encounter any "
+		"issues.");
 	return 0;
 }
 
